src/components/Form/TextInput.tsx

```diff
@@ -14,11 +14,14 @@ import { logger } from '../../utils/logger';
 import classnames from 'classnames';
 import styled from 'styled-components';
 import { disabledCursor } from '../utils/css';
+import { idGenerator } from '../../utils/uuid';
 
 const baseClassName = 'fi-text-input';
 const disabledClassName = `${baseClassName}--disabled`;
 const labelBaseClassName = `${baseClassName}_label`;
 const inputBaseClassName = `${baseClassName}_input`;
+const statusTextClassName = `${baseClassName}_statusText`;
```

Comment:
Should this be statusTextContainerClassName and statusTextClassName (since status text is not a label)?

Reply:
After the recent commit, these are now like in following:

https://github.com/vrk-kpa/suomifi-ui-components/blob/4240cc93b646b319c771e41b9c4887a0635d6c8a/src/components/Form/TextInput.tsx#L23-L25